.vscode/settings.json

```diff
@@ -10,9 +10,8 @@
     "**/.git": true,
     "**/.svn": true,
     "**/.hg": true,
-    "**/.DS_Store": true,
     "**/node_modules": true,
-    "build": true,
```

Comment:
why do we need to include the `build` output in vs code?

Reply:
My bad, leftover from debugging stuff.